packages/cli/src/databases/repositories/projectRelation.repository.ts

```diff
@@ -52,4 +52,13 @@ export class ProjectRelationRepository extends Repository<ProjectRelation> {
 			{} as Record<ProjectRole, number>,
 		);
 	}
+
+	async findUserIdsByProjectId(projectId: string): Promise<string[]> {
+		const rows = await this.find({
```

Comment:
we should update `.find` to accept a `distinct: boolean` to do the deduplication on the DB itself, and not in the app.